Sources/MapboxNavigation/IdleTimerManager.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import UIKit
+
+/// Manages UIApplication.shared.isIdleTimerDisabled
+final class IdleTimerManager {
+    final class Token {
```

Comment:
Maybe `Handle` is a more natural name for this structure. Similar to `CacheHandle`

Reply:
I end up with Cancellable similar to Combine.Cancellable concept. But couldn't use Combine due to deployment target limitations.

Reply:
`Combine.Cancellable`, so be it.